### PR TITLE
Build mart models

### PR DESCRIPTION
**Pull request by wasilaq**
image

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -32,7 +25,8 @@ clean-targets:         # directories to be removed by `dbt clean`
 # as tables. These settings can be overridden in the individual model files
 # using the `{{ config(...) }}` macro.
 models:
-  my_new_project:
-      # Applies to all files under models/example/
-      example:
-          materialized: view
+  dbt_tickets:
+    staging:
+      +materialized: view
```
Comment (dave-connors-3) [inline dbt_project.yml:30]: small comment -- the default materialization in dbt is a view, so you don't need this line! no harm in keeping it though

File: models/marts/core/fct_tickets.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
+with tickets as (
+
+    select
+      ticket_id,
+      status,
+      event_id
+    from {{ ref('stg_issued_tickets') }}
```
Comment (dave-connors-3) [inline models/marts/core/fct_tickets.sql]: again, this is a small comment, but generally we would write this as a stand alone import CTE to declare dependencies at the top of the model. [our style guide](https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md) has a good example a full model syntax if you're curious!

File: models/marts/core/fct_tickets.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      venue,
+      postal_code,
+      event_id
+    from {{ ref('stg_events') }}
+
+)
+
+
+select
+    ticket_id,
```
Comment (dave-connors-3) [inline models/marts/core/fct_tickets.sql]: another small style guide thing is including the table prefix for each field!

File: models/staging/ticket_tailor/stg_events.sql [hunk 4]
```diff
@@ -4,7 +4,7 @@ with events as (
       id as event_id,
       name as event_name,
       status,
-      private::boolean as is_private,
+      private::boolean as private_event,
```
Comment (dave-connors-3) [inline models/staging/ticket_tailor/stg_events.sql]: any reason for this change?

**Review by dave-connors-3 — COMMENTED** [review]
looks good! some small style guide comments, but nothing major. I know you said you might be making some updates, so feel free to re-request a review whenever you are ready! If you also can include a screenshot of the DAG in the comment section of the PR, that would be wonderful 🙂

File: models/staging/ticket_tailor/stg_events.sql (hunk 4 above)
Comment (wasilaq) [reply models/staging/ticket_tailor/stg_events.sql]: There was a reason but I don't remember what it was 😓

File: dbt_project.yml (hunk 1 above)
Comment (wasilaq) [reply dbt_project.yml:30]: keeping this in for consistency (so each folder has an explicitly mentioned materialization)

File: models/marts/core/fct_tickets.sql (hunk 3 above)
Comment (wasilaq) [reply models/marts/core/fct_tickets.sql]: fixed

File: models/marts/core/fct_tickets.sql (hunk 2 above)
Comment (wasilaq) [reply models/marts/core/fct_tickets.sql]: I believe I've fixed this correctly but not 100% sure

**Pull request merged**